rich-text.sh

```
@@ -1 +1 @@
-RUST_LOG=cosmic_text=debug,rich_text=debug cargo run --release --package rich-text -- "$@"
+RUST_LOG=fastui_cosmic=debug,rich_text=debug cargo run --release --package rich-text -- "$@"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)